src/createPackage/models/createPackageManager.ts

```diff
@@ -267,6 +262,24 @@ export class CreatePackageManager {
     return sanitized;
   }
 
+  private generateTileGroups(polygon: Polygon, footprint: Polygon | MultiPolygon, zoom: number): ITileRange[] | null {
+    const intersaction = intersect(polygon, footprint);
+    const tilesGroups: ITileRange[] = [];
+
+    if (intersaction === null) {
+      return null;
+    }
+
+    for (let i = 0; i <= zoom; i++) {
+      const zoomTilesGroups = new TileRanger().encodeFootprint(intersaction, i);
```

Comment:
protect with try catch
if error print error details

Reply:
same as the previous comment - we can only check if intersaction is null and throw new BadRequestError with those details as we did to the sanatized (uses the same intersaction func from turf)

Reply:
ok, but open an issue regarding this issue for later usage

Reply:
fixed